cylc/flow/exceptions.py

```diff
@@ -262,6 +262,13 @@ def __init__(self, platform):
         super().__init__(f'Unable to find valid host for {self.platform_n}')
 
 
+class NoPlatformsError(CylcError):
+    """None of the platforms of a given group were reachable."""
+    def __init__(self):
+        super().__init__(
+            'Unable to find valid platform.')
```

Comment:
```suggestion
            'Unable to find a reachable platform.')
```
(Platforms can be valid without necessarily being reachable at the moment?)